src/finance-page/finance-header/finance-header.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import './finance-header.css';
+import '../finance-page.css';
```

Comment:
Why are you importing this css?

Reply:
I use here full-with class for header and the styles for this class are declared in finance-page.css

Reply:
So put all styles you wanna share inside some `shared.css`, and import it in your App.